### PR TITLE
Fix new line .

### PR DESCRIPTION
**Pull request by YoussefEssalhi**
Fix : #604

### CODE REVIEW

File: src/inquirer/render/console/__init__.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def render(self, question, answers=None):
         try:
             return self._event_loop(render)
         finally:
-            print("")
+            print('', end='')
```
Comment (Cube707) [inline src/inquirer/render/console/__init__.py:40]: this is a nonsensical operation, use this instead
```suggestion
            pass
```

**Comment by Cube707** [comment]
I just realized that this also affects multiple prompts. The empty line is important to keep them seperate.

Original behavior:
image

Changed behavior:
image


This is definitely not an option to change!

File: src/inquirer/render/console/__init__.py (hunk 1 above)
Comment (YoussefEssalhi) [reply src/inquirer/render/console/__init__.py:40]: It prints an empty string without a newline hh

Comment (YoussefEssalhi) [reply src/inquirer/render/console/__init__.py:40]: It print an empty string without a newline hh

**Comment by Cube707** [comment]
> a custom parameter "separate" and be enabled by default
Yes this would be the optimal solution and there are multiple other cases where such a configuration would be useful. For example see the `readchar` libary that implements such a pattern with its `config`, which would be useful to hold as a part of `inquirers` config. Another idea is to allow for the deletion of the question from the screen after its answerd. and many more.

But Implementing this pattern would be quite the task and I simply don't have the time.

I do not think this would be useful to have as a argument when construction a prompt, they are already overloaded with options as is...

**Pull request closed without merging**